.github/workflows/spa-build.yml

```
@@ -0,0 +1,185 @@
+jobs:
+  resolve-config:
+    name: Resolve ${{ github.event.inputs.pipeline_name }} config
+    runs-on: ubuntu-latest
+    outputs:
+      work_dir:   ${{ steps.setenv.outputs.work_dir }}
+      image_name: ${{ steps.setenv.outputs.image_name }}
+      dockerfile: ${{ steps.setenv.outputs.dockerfile }}
+      tag:        ${{ steps.tag.outputs.tag }}
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)